src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1841,27 +1836,22 @@ private FeedbackSessionResultsBundle getFeedbackSessionResultsForUserWithParams(
                             if (role == UserRole.INSTRUCTOR) {
                                 instructor = instructorsLogic.getInstructorForEmail(courseId, userEmail);
                             }
-                            boolean isVisibleResponse = isResponseVisibleForUser(userEmail,
-                                    role, null, null, response, question, instructor);
+                            boolean isVisibleResponse = isResponseVisibleForUser(userEmail, role, null, null, response,
+                                                                                 question, instructor);
                             if (isVisibleResponse) {
                                 responses.add(response);
-                                addEmailNamePairsToTable(emailNameTable, response,
-                                        question, roster);
-                                addEmailLastNamePairsToTable(emailLastNameTable, response,
-                                        question, roster);
-                                addEmailTeamNamePairsToTable(emailTeamNameTable,
-                                        response,
-                                        question, roster);
-                                addVisibilityToTable(visibilityTable, question,
-                                        response, userEmail, role, roster);
+                                addEmailNamePairsToTable(emailNameTable, response, question, roster);
+                                addEmailLastNamePairsToTable(emailLastNameTable, response, question, roster);
+                                addEmailTeamNamePairsToTable(emailTeamNameTable, response, question, roster);
+                                addVisibilityToTable(visibilityTable, question, response, userEmail, role, roster);
                             }
                         }
                     }
                 }
             }
 
-            addSectionTeamNamesToTable(sectionTeamNameTable, roster, courseId, userEmail, role,
-                                       feedbackSessionName, section);
+            addSectionTeamNamesToTable(sectionTeamNameTable, roster, courseId, userEmail,
+                                       role, feedbackSessionName, section);
```

Comment:
Here, it can be:
```java
addSectionTeamNamesToTable(sectionTeamNameTable,
        roster, courseId, userEmail, role, feedbackSessionName, section);
```
or
```java
addSectionTeamNamesToTable(
        sectionTeamNameTable, roster, courseId, userEmail, role, feedbackSessionName, section);
```